module-apps/application-call/windows/CallWindow.cpp

```diff
@@ -154,6 +168,23 @@ namespace gui
         }
     }
 
+    void CallWindow::updateEnteredNumber(const char &text, bool isVisible)
```

Comment:
This entire logic should IMO be put in presenter instead of view, now it breaks MVP model rules - business logic is present in view part.